styles/responsive.css

```diff
@@ -0,0 +1,11 @@
+@media screen and (max-width: 1000px) {
+  .header > .col-lg {
```

Comment:
I would suggest using a custom class here rather than applying styling to the `col-lg` class, since that originates from Bootstrap :grin: